src/components/Tag.tsx

```diff
@@ -1,76 +1,41 @@
 import React from "react"
-import styled from "@emotion/styled"
-import Icon from "./Icon"
-
-export interface IIsActive {
-  isActive: boolean
-}
-
-const StyledTag = styled.button<IIsActive>`
-  display: flex;
-  align-items: center;
-  padding: 4px 8px;
-  margin-bottom: 0.5rem;
-  margin-right: 0.5rem;
-  background: radial-gradient(
-    46.28% 66.31% at 66.95% 58.35%,
-    rgba(127, 127, 213, 0.2) 0%,
-    rgba(134, 168, 231, 0.2) 50%,
-    rgba(145, 234, 228, 0.2) 100%
-  );
-  border-radius: 4px;
-  text-transform: uppercase;
-  font-size: 0.875rem;
-  box-shadow: ${(props) =>
-    props.isActive ? 0 : props.theme.colors.tableBoxShadow};
-  border: 1px solid
-    ${(props) =>
-      props.isActive
-        ? props.theme.colors.primary300
-        : props.theme.colors.white800};
-  cursor: pointer;
-  color: ${(props) => props.theme.colors.text};
-  svg {
-    fill: ${(props) => props.theme.colors.text};
-  }
-  opacity: ${(props) => (props.isActive ? 1 : 0.7)};
-
-  &:hover {
-    color: ${(props) => props.theme.colors.primary};
-    border: 1px solid ${(props) => props.theme.colors.text200};
-    opacity: 1;
-    svg {
-      fill: ${(props) => props.theme.colors.primary};
-    }
-  }
-`
-
-const StyledIcon = styled(Icon)`
-  margin-left: 1em;
-`
-
-export interface IProps {
+import {
+  Tag as ChakraTag,
+  TagRightIcon,
+  TagLabel,
+  TagProps,
+} from "@chakra-ui/react"
+import { MdAdd, MdClose } from "react-icons/md"
+
+export interface IProps extends TagProps {
   name: string
-  onSelect: (tagName: string) => void
+  onChoose: (tagName: string) => void
   value: string
   isActive: boolean
   shouldShowIcon: boolean
 }
 
 const Tag: React.FC<IProps> = ({
   name,
-  onSelect,
+  onChoose,
   value,
   isActive = true,
   shouldShowIcon = true,
 }) => {
-  const iconName = isActive ? "close" : "add"
-
   return (
-    <StyledTag onClick={() => onSelect(value)} isActive={isActive}>
-      {name}
-      {shouldShowIcon && <StyledIcon name={iconName} size="18" />}
-    </StyledTag>
+    <ChakraTag
+      onClick={() => onChoose(value)}
```

Comment:
I would call it `onClick` to be aligned with the Chakra naming since we are exporting a `ChakraTag` component.
```suggestion
      onClick={() => onClick(value)}
```

Reply:
Agree, much more sensible